stdlib/public/stubs/Stubs.cpp

```diff
@@ -128,6 +128,26 @@ static locale_t getCLocale() {
 }
 #endif
 
+#ifdef __ANDROID__
+#define uselocale(a) (nullptr)
+static double swift_strtod_l(const char* a, char** b, locale_t c) {
+	return strtod(a, b);
+}
+static float swift_strtof_l(const char* a, char** b, locale_t c) {
+	return strtof(a, b);
```

Comment:
Here I've decided to:

1. Remove the `#define uselocale(a)`, since that function is available on API 21 (which this work is now based upon).
2. Convert tabs to spaces.
3. Preserve the definitions of `strtod_l` and friends here, since they're unavailable on Android.

Does that sound reasonable? Of course we can always improve upon this in the future.